packages/components/src/internal/components/domainproperties/models.spec.ts

```diff
@@ -51,6 +55,128 @@ import {
     DOMAIN_FIELD_FULLY_LOCKED,
 } from './constants';
 
+const gridDataConst = [
```

Comment:
Wasn't quite sure where to put this comparison constant—whether to put it in a separate file or keep it closer to the test within this file